IPython/html/nbextensions.py

```diff
@@ -96,16 +127,29 @@ def install_nbextension(files, overwrite=False, symlink=False, ipython_dir=None,
         Not allowed with URLs or archives. Windows support for symlinks requires
         Vista or above, Python 3, and a permission bit which only admin users
         have by default, so don't rely on it.
-    ipython_dir : str [optional]
-        The path to an IPython directory, if the default value is not desired.
-        get_ipython_dir() is used by default.
+    user : bool [default: False]
+        Whether to install to the user's .ipython/nbextensions directory.
+        Otherwise do a system-wide install (e.g. /usr/local/share/jupyter/nbextensions).
+    prefix : str [optional]
+        Specify install prefix, if it should differ from default (e.g. /usr/local).
+        Will install to prefix/share/jupyter/nbextensions
+    nbextensions : str [optional]
```

Comment:
Just a minor, but I find the `nbextensions` arg name a little confusing... could not be modified to something pointing to the abs path idea?